e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -507,6 +451,102 @@ describe("scenarios > dashboards > filters > auto apply", () => {
         undoToast().should("not.exist");
       });
     });
+
+    describe("full-app embeddings", () => {
```

Comment:
Add 3 tests
1. Similar to normal dashboards: Can apply filters via `Apply` button.
2. Show auto-apply filter toast for users with dashboard's write access.
3. Don't show auto-apply filter toasts when logged in with users without dashboard's write access.